crypto/nrf_cc310_platform/CHANGELOG.rst

```diff
@@ -9,6 +9,34 @@ Changelog - nrf_cc3xx_platform
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_platform - 0.9.17
+***************************
+
+New version of the library with the following features:
+
+* Removed mbedtls_hardware_poll limitation on buffer size, it now accepts any buffer size.
```

Comment:
Same formatting note as above, but I'm not sure what this hardware_poll falls under.

```suggestion
* Removed mbedtls_hardware_poll limitation on buffer size. It now accepts any buffer size.
```

Should be two sentences. But what does 'it' refer to? The library?

Reply:
"It" is mbedtls_hardware_poll , which is a function.

Reply:
Removed limitation in ``mbedtls_hardware_poll`` function on buffer size. It now accepts any buffer size.

Reply:
Somehow, when it's formatted, it's easier for me to understand that 'it' refers to the function. looks good now with that.